### PR TITLE
React-Redux Project - BookStore - API Work

### PR DESCRIPTION
**Pull request by jmontanholi**
## In this feature branch I refactored the code to use the API to:
* #### Add books directly to the API
* #### Remove books directly from the API
* #### Fetch the books from the API to show them in the UI

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -9,7 +10,13 @@ const reducer = combineReducers({
...
+  applyMiddleware(ReduxThunk, logger),
 );
 
 export default store;
+// https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/YjNKpt0nstX4A8fNqIZ4/books
+// {
+//   "item_id": "item1",
+//   "title": "The Great Gatsby",
+//   "category": "Fiction"
+// }
```
Comment (christianotieno) [inline src/redux/configureStore.js]: - [x] We always try to ensure that every work that gets submitted gets to mirror a close if not final work and is ready for deployment, and that will require minimal changes if needed. As a result, one of the areas is to always ensure our [code is clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) first and foremost before anything else.  Kindly remove these commented lines code as they tend to make the codebase ugly and untidy.

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hi @jmontanholi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (jmontanholi) [reply src/redux/configureStore.js]: Thanks! I'll be honest, you scared me a bit with this text, I thought I had done something extremely wrong with the code. Hahahahaha.

Comment (bellom) [reply src/redux/configureStore.js]: @jmontanholi don't be scared, that is what we call a professional way of correcting someone. 👍 

@chris 👍 🙌

**Review by bellom — APPROVED** [review]
## Hi @jmontanholi  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**